zebra-rpc/src/methods.rs

```diff
@@ -601,6 +608,28 @@ where
                     _ => unreachable!("unmatched response to a block request"),
                 }
             } else if verbosity == 1 {
+                // # Performance
+                //
+                // This RPC is used in `lightwalletd`'s initial sync of 2 million blocks,
+                // so it needs to load all its fields very efficiently.
```

Comment:
Question: Do lightwalletd uses both verbosity in the initial 2 million blocks ?

Reply:
Yes, unfortunately!